components/common/src/ome/api/Search.java

```diff
@@ -158,7 +158,7 @@
 
     /**
      * Returns the current case sensitivity setting. If
-     * {@link #setCaseSentivice()} has not been called, the
+     * {@link #setCaseSentivice(boolean)} has not been called, the
```

Comment:
Agreed. This work is meant to be develop only indeed. Will open a follow-up PR once this is in and have @joshmoore check it is not breaking anything.